platforms/ios/src/TangramMap/TGSceneUpdateStatus.mm

```diff
@@ -0,0 +1,55 @@
+//
+//  TGSceneUpdateStatus.h
+//  TangramMap
+//
+//  Created by Karim Naaji on 3/07/17.
+//  Copyright (c) 2017 Mapzen. All rights reserved.
+//
+
+
+#import "TGSceneUpdateStatus.h"
+#import "TGSceneUpdate.h"
+#import "TGHelpers.h"
+
+@interface TGSceneUpdateStatus ()
+
+@property (assign, nonatomic) TGSceneUpdate* sceneUpdate;
+@property (assign, nonatomic) TGSceneUpdateError error;
+
+@end
+
+@implementation TGSceneUpdateStatus
+
+- (instancetype)initWithSceneUpdate:(TGSceneUpdate *)sceneUpdate sceneUpdateError:(TGSceneUpdateError)error
+{
+    self = [super init];
+
+    if (self) {
+        self.sceneUpdate = sceneUpdate;
+        self.error = error;
+    }
+
+    return self;
+}
+
++ (NSArray<TGSceneUpdateStatus *> *)sceneUpdateStatuses:(const std::vector<Tangram::SceneUpdateStatus>&)updateStatuses
+{
+    NSMutableArray* nsUpdateStatuses = [[NSMutableArray alloc] initWithCapacity:(NSUInteger)updateStatuses.size()];
```

Comment:
`updateStatuses` is a reference though, which should theoretically be initialized and not `null`, not even testable against it (unless you're doing something strange and dereference something already `null`), we're safe here.